actions/publish-npm/main.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+cd llama-stack-client-typescript
+if [[ "$VERSION" == *"rc"* ]]; then
+  yarn version --new-version $VERSION -m "Bump version to $VERSION" --no-git-tag-version
```

Comment:
hm, shouldn't you do this always because you are manually specifying the new version yourself?

Reply:
if "rc", then we apply the `--no-git-tag-version` flag to avoid creating too many rc tags